webdriver/tests/sessions/new_session/default_values.py

```diff
@@ -36,21 +36,13 @@ def test_desired(new_session):
         resp, _ = new_session({"desiredCapbilities": {}})
 
 
-def test_required(new_session):
```

Comment:
I think that removing these makes the testsuite less useful (in particular it makes Geckodriver pass more tests despite having known non-conformance here).